dist/21-526EZ-schema.json

```diff
@@ -2,7 +2,473 @@
+                "state": {
+                  "type": "string",
+                  "maxLength": 51
+                }
+              }
+            }
+          ],
+          "properties": {
+            "addressLine1": {
+              "type": "string"
+            },
+            "addressLine2": {
+              "type": "string"
+            },
+            "addressLine3": {
+              "type": "string"
+            },
+            "city": {
+              "type": "string"
+            },
+            "zipFirstFive": {
+              "type": "number",
+              "minLength": 5,
+              "maxLength": 5
+            },
+            "zipLastFour": {
+              "type": "number",
+              "minLength": 4,
+              "maxLength": 4
+            },
+            "militaryPostOfficeTypeCode": {
+              "type": "string"
+            },
+            "militaryStateCode": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string",
+              "enum": [
+                "international",
+                "domestic",
+                "military"
+              ]
+            }
+          }
+        },
+        "primaryPhone": {
+          "$ref": "#/definitions/fullPhone"
+        },
+        "forwardingAddress": {
+          "type": "object",
+          "oneOf": [
+            {
+              "properties": {
+                "country": {
+                  "not": {
+                    "type": "string",
+                    "enum": []
+                  }
+                },
+                "state": {
+                  "type": "string",
+                  "maxLength": 51
+                }
+              }
+            }
+          ],
+          "properties": {
+            "addressLine1": {
+              "type": "string"
+            },
+            "addressLine2": {
+              "type": "string"
+            },
+            "addressLine3": {
+              "type": "string"
+            },
+            "city": {
+              "type": "string"
+            },
+            "zipFirstFive": {
+              "type": "number",
+              "minLength": 5,
+              "maxLength": 5
+            },
+            "zipLastFour": {
+              "type": "number",
+              "minLength": 4,
+              "maxLength": 4
+            },
+            "militaryPostOfficeTypeCode": {
+              "type": "string"
+            },
+            "militaryStateCode": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string",
+              "enum": [
+                "international",
+                "domestic",
+                "military"
+              ]
+            },
+            "efctvDt": {
+              "$ref": "#/definitions/date"
+            }
+          }
+        },
+        "homelessness": {
+          "type": "object",
+          "properties": {
+            "hasPointOfContact": {
+              "type": "boolean"
```

Comment:
This is one of the things I wanted to remove but looks to still be in at the moment.  Seems like the concept of "has point of contact" can be deduced by if `pointOfContact` is `null` or not.